examples/how-to-simple-python.md

```diff
@@ -49,16 +49,17 @@ def get_access_token():
 
 ### Make an API call
 
-Now we are ready to make an API call.
+Now we are ready to make an API call. 
+To run the script against with your application credentials replace client_id, client_secret, environment url and process id.
```

Comment:
I think it should be, "To run the script with your application credentials, replace client_id, client_secret, environment url and process id."

Reply:
@hal9000-hub this is fixed.